test/common_plat/performance/odp_l2fwd_run.sh

```diff
@@ -30,7 +30,7 @@ PATH=$TEST_DIR:$TEST_DIR/../../../example/generator:$PATH
 # exit codes expected by automake for skipped tests
 TEST_SKIPPED=77
 
-PLATFORM_VALIDATION=${TEST_SRC_DIR}/../../$ODP_PLATFORM/validation
+PLATFORM_VALIDATION=${TEST_SRC_DIR}/../../../platform/$ODP_PLATFORM/test/validation
```

Comment:
Checkpatch flags this line as being > 80 chars. Can it not be split into two lines or otherwise shortened?

Reply:
Not in a clean way

Reply:
What's wrong with factoring this like:
```
VALIDATION_TESTDIR=platform/$ODP_PLATFORM/test/validation
PLATFORM_VALIDATION=${TEST_SRC_DIR}/../../$VALIDATION_TESTDIR
```
That seems to solve the line length problem cleanly.

Reply:
just for the sake of 80 chars limit? Fine, I will update this.

Reply:
@Bill-Fischofer-Linaro updated